beamer/tests/agent/unit/test_challenge.py

```diff
@@ -99,7 +100,7 @@ def test_join_false_claim_challenge_only_when_unfilled(filler):
 def test_optimism_prove(mocked_relayer_call):
     timestamp = int(time.time())
     mocked_relayer_call.return_value = str(timestamp)
-    op_chain_id = ChainId(901)
+    op_chain_id = BEAMER_CONFIG.chains["optimism"]["local"].id
```

Comment:
I think instead of that, which requires the chain descriptor data to be public, we should have an API for queries like this. For example,

```python
descriptor = beamer.chains.find_chain_descriptor(ecosystem_name="optimism", name="local")
assert descriptor is not None
op_chain_id = descriptor.id
```